### PR TITLE
HEEDLS-681 LH SSO Account Link page test fixes

### PR DESCRIPTION
**Pull request by wilkinsm**
### JIRA link
[https://softwiretech.atlassian.net/browse/HEEDLS-681](https://softwiretech.atlassian.net/browse/HEEDLS-681)

### Description

- Update header text
- Fix grammar in content
- Merge `Signposting` namespaces

### Screenshots
Highlighted in yellow are the changes made.
image

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [x] Run the formatter and made sure there are no IDE errors.
- [x] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [x] Manually tested my work with and without JavaScript. Full manual testing guidelines can be found here: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/6703648740/Testing
- [ ] Updated/added documentation in Swiki and/or Readme. Links (if any) are below:
  - [doc_1_here](link_1_here)
- [ ] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken.
- [x] Scanned over my own MR to ensure everything is as expected.

### CODE REVIEW

File: DigitalLearningSolutions.Web/Views/Signposting/LinkLearningHubSso.cshtml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-@using DigitalLearningSolutions.Web.ViewModels.SignpostingSso
+@using DigitalLearningSolutions.Web.ViewModels.Signposting
 @model LinkLearningHubViewModel
 
 @{
-    ViewData[LayoutViewDataKeys.Title] = "Recommended Learning Resources - Account linked";
-    ViewData[LayoutViewDataKeys.Application] = "Recommended Learning Resources";
+    ViewData[LayoutViewDataKeys.Title] = "Learning Hub Account Linked";
+    ViewData[LayoutViewDataKeys.Application] = "Digital Learning Solutions Learning Portal";
     ViewData[LayoutViewDataKeys.HeaderPrefix] = "";
```
Comment (SteveJacksonSoft) [inline DigitalLearningSolutions.Web/Views/Signposting/LinkLearningHubSso.cshtml:8]: I think you can just remove these lines and [SetDlsSubApplication] will achieve the same things.

**Review by SteveJacksonSoft — CHANGES_REQUESTED** [review]
Just a change to view data that could help.

File: DigitalLearningSolutions.Web/Views/Signposting/LinkLearningHubSso.cshtml (hunk 1 above)
Comment (wilkinsm) [reply DigitalLearningSolutions.Web/Views/Signposting/LinkLearningHubSso.cshtml]: Nice, done.

Comment (SteveJacksonSoft) [reply DigitalLearningSolutions.Web/Views/Signposting/LinkLearningHubSso.cshtml]: (You can skip TL review when peer-review is done.)

**Review by SteveJacksonSoft — APPROVED** [review]
Noice

File: DigitalLearningSolutions.Web/Views/Signposting/LinkLearningHubSso.cshtml [hunk 2]
```diff
@@ -1,11 +1,9 @@
 ﻿@using DigitalLearningSolutions.Web.Helpers
-@using DigitalLearningSolutions.Web.ViewModels.SignpostingSso
+@using DigitalLearningSolutions.Web.ViewModels.Signposting
 @model LinkLearningHubViewModel
 
 @{
-    ViewData[LayoutViewDataKeys.Title] = "Recommended Learning Resources - Account linked";
-    ViewData[LayoutViewDataKeys.Application] = "Recommended Learning Resources";
-    ViewData[LayoutViewDataKeys.HeaderPrefix] = "";
+    ViewData[LayoutViewDataKeys.Title] = "Learning Hub Account Linked";
```
Comment (livzorn) [inline DigitalLearningSolutions.Web/Views/Signposting/LinkLearningHubSso.cshtml:6]: Just wondering about the capitalization here, I think we usually only capitalize the first letter in titles so maybe "Learning Hub account linked"?

**Review by livzorn — CHANGES_REQUESTED** [review]

File: DigitalLearningSolutions.Web/Views/Signposting/LinkLearningHubSso.cshtml (hunk 2 above)
Comment (wilkinsm) [reply DigitalLearningSolutions.Web/Views/Signposting/LinkLearningHubSso.cshtml:6]: Funnily that was what I was going to go with until I saw this style elsewhere such as:
https://github.com/TechnologyEnhancedLearning/DLSV2/blob/master/DigitalLearningSolutions.Web/Views/Signposting/LearningHubLoginWarning.cshtml#L6
Jumping around the site, it looks like there is some inconsistency on the approach here.
I will update this one as per your recommendation.

Comment (wilkinsm) [reply DigitalLearningSolutions.Web/Views/Signposting/LinkLearningHubSso.cshtml]: Thats done now

**Review by livzorn — APPROVED** [review]

**Pull request merged**